### DISCUSSION TITLE
SSH proxy support?

### DISCUSSION THREAD

**Discussion by MrCroxx** [GitoxideLabs/gitoxide#945] [Q&A] [answered]
Hi, I tried gitoxide on https and it's amazing. I also need SSL proxy in my environment. I searched document and issue but I didn't found a configuration for it. Did I miss something or it's not supported yet? P.S. I've set SSL config in my `~/.ssh/config` and my git works.

**Comment by Byron** [comment, marked as answer]
`gitoxide` uses the `ssh` program, just like `git`, which respects the `~/.ssh/config`. Maybe there is a specific problem you can share? Maybe #933 is relevant to you.

**Reply by MrCroxx** [reply]
Thanks for your reply. #933 is related that I cloned my repo with an alias that has a dot in its directory name.

**Reply by Byron** [reply]
Does that mean your problem is actually #933? If you change to the url to not use an alias, will it work as expected? In any case, it looks like #933 should be fixed sooner than later.
